src/clj/commiteth/eth/core.clj

```diff
@@ -77,8 +77,11 @@
 (defn eth-gasstation-gas-price
   []
   (let [data (json-api-request "https://ethgasstation.info/json/ethgasAPI.json")
-        avg-price (-> (get data "average")
-                      bigint)
+        avg-price (max
+                    (-> (get data "average")
+                        bigint)
+                    (-> (get data "average_calc")
+                        bigint))
```

Comment:
I'm not sure what the difference between these two is. Tried finding documentation around the ethgasstation.info API used here but couldn't. If you're aware of this kind of documentation please add :-)

Reply:
It's not in the documentation unfortunately. It seems that the only difference between two is that `average_calc` is calculated from a larger time period than `average`. Added a doc string to the function explaining that.